README.md

```diff
@@ -1,70 +1,70 @@
-# KubeAI Application Nucleus for edge (KAN)
+# KubeAI Application Nucleus for edge (SYMPHONY)

```

Comment:
Maybe I was clear - we are renaming backend to Symphony, not the KAN project itself. KAN project (the portal and ai inference container) is still KAN. Only the backend is Symphony.